landing-page.css

```diff
@@ -0,0 +1,111 @@
+:root{
+    --landing-page-bg-color:#fff8e7;
+    --landing-page-nav-links-color:#71bfff;
+}
+
+.whole-content{
+    background-color: var(--landing-page-bg-color);
+    font-family: 'Exo',sans-serif;
+    display: flex;
+    flex-direction: column;
+}
+/* NAVBAR RELATED STYLING OF LANDING PAGE */
+.site-nav{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding:0.7rem;
+    flex-wrap: wrap;
+    color:var(--landing-page-nav-links-color);
+    box-shadow:17px 22px 23px 1px rgb(34,219,175);
```

Comment:
i browsed your landing page preview, you have applied a high box shadow, you can try to reduce the values, may be it will then look more appealing(just a suggestion).
also, use color variable here!!